src/app/users/users-achievements/users-achievements.component.html

```diff
@@ -52,9 +52,9 @@ <h3 i18n>My References</h3>
         <mat-list>
           <mat-list-item class="mat-list-item-word-wrap" *ngFor="let reference of achievements.references">
             <h4 mat-line>{{reference.name}}</h4>
-            <p mat-line>{{reference.relationship}}</p>
-            <p mat-line>{{reference.phone}}</p>
-            <p mat-line>{{reference.email}}</p>
+            <p mat-line>Relationship:<span class="margin-lr-3">{{reference.relationship}}</span></p>
```

Comment:
We shouldn't use CSS styling to replace a space in the text.  Please update to use a space.

Reply:
I've replaced it with a space.